eva-server/src/main/java/uk/ac/ebi/eva/server/ws/VariantWSServer.java

```diff
@@ -131,6 +131,76 @@ public QueryResponse getVariantById(@PathVariable("variantId") String variantId,
         return setQueryResponse(queryResult);
     }
 
+    @RequestMapping(value = "/{variantId}/infoByList", method = RequestMethod.GET)
```

Comment:
Due variantId being a list, I'm not sure if this is the better way of defining the path. I Would use a request parameter insted, @cyenyxe any preferences?

Reply:
There is not a lot of consensus about how to do this in the REST community. Given that it is a way to specify multiple resources and it also is how we already do it, I would say we can leave it as is.

Reply:
There is no consensus on this but the use on comma on the parameters is better supported than in the path.

Reply:
Discussed offline and for the sake of consistency with the rest of the existing API we will leave it as is, but I would recommend changing the endpoint the value to `/{variantIds}`